tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py

```diff
@@ -94,10 +94,18 @@ def _test(self, model, ref_net, kind, ie_device, precision, ir_version, infer_ti
             fw_res = (fw_res,)
 
         output_list = list(infer_res.values())
-        assert len(fw_res) == len(
-            output_list), f'number of outputs not equal, {len(fw_res)} != {len(output_list)}'
+
+        flatten_fw_res = []
+        for res_item in fw_res:
+            # if None is at output we skip it
+            if res_item is None:
```

Comment:
can you please elaborate in what cases the result can be `None`?

Reply:
the most popular case is embedding loss function calculation in model, which used only in training stage. When we move model to eval mode and does not provide annotation, it is not calculated and return by default None.
As example, https://github.com/pytorch/vision/blob/main/torchvision/models/inception.py#L170

Reply:
can we add such comment into the code?

Reply:
Added comment